src/stories/Blocks/material-manifestation-item/material-manifestation-item.scss

```diff
@@ -83,7 +83,7 @@
       align-items: center;
       justify-self: end;
     }
-    button {
+    button:first-of-type {
```

Comment:
Isn't it fragile to take for granted that the design is determined by the order of the buttons?
I would give it a class but if there is a strong concept/methodology behind it I am open for changing my mind :)

Reply:
That's a fair point that I haven't thought of! :) I'll change that.